src/game/__init__.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import logging
+import random
+from typing import List, Tuple
+from uuid import UUID
+
+from .player import Player
+
+
+class WaitingRoom:
+    def __init__(self) -> None:
+        self.players: List[Player] = []
+
+    def draw_two_players_to_game(self) -> Tuple[Player, Player]:
+        random.shuffle(self.players)
+        players = (self.players.pop(), self.players.pop())
+        for p in players:
+            p.in_game = True
+        return players
+
+    def join(self, player: Player):
+        self.players.append(player)
+        logging.info("Player '%s' joined waiting room" % player.nick)
+
+
+class Game:
+    def __init__(self, players: Tuple[Player, Player]) -> None:
+        self.players: Tuple[Player, Player] = players
+
+    def _get_nicks_of_players(self) -> Tuple[str, str]:
+        return self.players[0].nick, self.players[1].nick
+
+    async def play(self) -> None:
+        logging.info(
+            "Start game of players: '%s' and '%s'" % self._get_nicks_of_players()
+        )
+        # Game
+        self._end_game_for_players()
+        logging.info(
+            "Finish game of players: '%s' and '%s'" % self._get_nicks_of_players()
+        )
+
+    def _end_game_for_players(self) -> None:
+        for p in self.players:
+            p.in_game = False
+
+
+class GameApp:
+    def __init__(self) -> None:
+        self.players: List[Player] = []
+        self.waiting_room: WaitingRoom = WaitingRoom()
+
+    def add_player(self, nick: str) -> UUID:
+        exist_player: Player = self.get_player_by_nick(nick)
```

Comment:
Może _player_with_same_nick_, albo _existing_player_. Mogą też być problemy potem jeśli dostaniemy id innego gracza zamiast błędu, ale to zależy jak będziemy tego używać, nie wiem jak jest lepiej :smiley: